### PR TITLE
Move recommendation to 18.04 #1030

### LINKED ISSUE #1030: Move recommendation to 18.04
We still recommend 16.04 in the docs. 18.04 shoud be used.

### PR DESCRIPTION
**Pull request by fgsl**
Fix #1030

### CODE REVIEW

**Comment by rullzer** [comment]
But are the installation steps actually the same on 18.04 ?

**Comment by J0WI** [comment]
I think we should switch to version independent packages in the installation guide (e.g. `libapache2-mod-php` instead of `libapache2-mod-php7.2`, `php-curl` instead of `php7.2-curl` etc.).
There are much more of these in `source_installation.rst`.

**Comment by fgsl** [comment]
@J0WI, I suggest to keep section "Example installation on Ubuntu 16.04 LTS server" and add right after "Example installation on Ubuntu 18.04 LTS server", but in other issue.  What do you think?

**Comment by J0WI** [comment]
I fine with this. But after switching to version independent packages names there shouldn't be a big difference between the installation steps on 16.04 and 18.04.

**Comment by skjnldsv** [comment]
@fgsl let's keep it like that. Simpler is cleaner :)
Could you make sure the various path are the same from 16.04 to 18.04?

**Comment by fgsl** [comment]
@skjnldsv, in Ubuntu 18.04, PHP packages are in version 7.2 (for instance, libapache2-mod-php7.2). However, the virtual package, without version, is the same. If suffix "7.0" is removed from current Ubuntu 16.04 installation section, it's enough to change title to "Example installation on Ubuntu 16.04 LTS or 18.04 LTS server".

**Comment by skjnldsv** [comment]
@fgsl Well, I think we should only suggest 18.04 and php7.2 then.
Let's update all the path and references ? :)

**Comment by fgsl** [comment]
@rullzer, @J0WI, what do you think? 

1) I merge this pull request, only with recommendation change?
2) I update this PR, including change in Ubuntu install steps?
3) I change Ubuntu install steps replacing packages of 16.04 by 18.04 (like to move php 7.0 to php 7.2) or to let generic steps to both versions, using virtual packages?

**Comment by skjnldsv** [comment]
Please change Ubuntu install steps replacing packages of 16.04 by 18.04 (like to move php 7.0 to php 7.2).

**Comment by fgsl** [comment]
@skjnldsv, I have changed Ubuntu install steps.

File: admin_manual/installation/source_installation.rst [hunk 1]
```diff
@@ -183,9 +183,9 @@ Or you can use .deb packages to install the required and recommended modules for
...
 
-    apt-get install apache2 mariadb-server libapache2-mod-php7.0
-    apt-get install php7.0-gd php7.0-json php7.0-mysql php7.0-curl php7.0-mbstring
-    apt-get install php7.0-intl php7.0-mcrypt php-imagick php7.0-xml php7.0-zip
+    apt-get install apache2 mariadb-server libapache2-mod-php7.2
+    apt-get install php7.2-gd php7.2-json php7.2-mysql php7.2-curl php7.2-mbstring
+    apt-get install php7.2-intl php-imagick php7.2-xml php7.2-zip
 
 * This installs the packages for the Nextcloud core system.
   ``libapache2-mod-php7.0`` provides the following PHP extensions: ``bcmath bz2
```
Comment (J0WI) [inline admin_manual/installation/source_installation.rst]: This should also be changed to `libapache2-mod-php7.2`

Comment (fgsl) [reply admin_manual/installation/source_installation.rst]: Right. I changed it.

**Pull request merged**